test/krm/behaviors/generate-resources.yaml

```diff
@@ -10,6 +10,4 @@ metadata:
           # otherwise, path should be relative to manifest files, like
           # path: ../../../ksops
 files:
-  # NOTE: File is relative to the /base manifest! 
```

Comment:
I believe this is actually a bug that was fixed in https://github.com/kubernetes-sigs/kustomize/pull/4654

Reply:
Nice! You're right